Assets/Scripts/Menus/MenuMusic.cs

```diff
@@ -9,10 +9,7 @@ public class MenuMusic : MonoBehaviour
     // Singleton design pattern to avoid audio duplication
     private static MenuMusic instance = null;
     public static MenuMusic Instance { get { return instance; } }
-
-    // Override via the editor as needed if build settings change
-    public int menuIndex = 0;
-    public int controlsIndex = 1;
+    public static AudioSource GetMusic() { return Instance.GetComponent<AudioSource>(); }
```

Comment:
lol I wasn't aware of this method. I just added an audio source for my level change scene.

Reply:
Not sure what you mean, that's what I did as well :)

Reply:
Basically, I didnt use this method to place the music on my scene (duplicating audio source).

Reply:
I should clarify :D There's no duplication; this is just returning a reference to the audio component.